tests/migration_test.go

```diff
@@ -2948,25 +2951,25 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				}
 			}
 
-			getVmisNamespace := func(vmi *v1.VirtualMachineInstance) *k8sv1.Namespace {
```

Comment:
This change does not seem to be related to this commit.

Reply:
I had to refactor the test to have it passing, and as a result the function is not needed anymore.

Reply:
ack